client/app/config/dashboard-grid-options.js

```diff
@@ -0,0 +1,35 @@
+const dashboardGridOptions = {
+  columns: 6,
+  pushing: true,
+  floating: true,
+  swapping: true,
+  width: 'auto',
+  colWidth: 'auto',
+  rowHeight: 50,
+  margins: [15, 15],
+  outerMargin: false,
+  sparse: false,
+  isMobile: false,
+  mobileBreakPoint: 1200,
```

Comment:
Let's use 800 here. 

In the future we might want to implement a better logic here, which will take into account the widgets sizes when stacking them. So small ones won't get stacked, but that's for the future.